app/views/course/courses/show.html.slim

```diff
@@ -10,3 +10,7 @@
   h2 = t('.instructors')
   div
     = render partial: 'layouts/user', collection: current_course.instructors.map(&:user)
+  - if current_user.courses.include?(current_course)
+    h2 = t('.announcements')
+    div
+      = render partial: 'course/announcements/announcement', collection: current_course.announcements.find_all(&:currently_active?)
```

Comment:
You can implement Course::Announcement#to_partial_path and you won't need to specify the partial.